src/accounts/remove/remove-account-handler.js

```diff
@@ -0,0 +1,202 @@
+const HttpClient = require('../../core/client/http-client');
+const ApiExtract = require('../../core/utils/apiExtract');
+const shouldDeleteDefaultAccountQuestion =
+  ' is your workspace account. Are you sure you want to remove it from Slack?';
+const shouldDeleteAccountWithCurrentChannels = ' is used in these channels:';
+const areYouSure = ' Are you sure you want to remove it from Slack?';
+const shouldDeleteAccount = 'Are you sure you want to remove ';
+
+function getMessageWithButtons(prefix, question, suffix) {
+  return {
+    attachments: [
+      {
+        title: '',
+        text: prefix + question + suffix,
```

Comment:
way not just pass text if you passing 3 arguments and concat them in one line
so just pass one argument call text